packages/devextreme-scss/scss/widgets/base/chat/layout/chat-avatar/_index.scss

```diff
@@ -1,7 +1,18 @@
 .dx-chat-message-avatar {
+  position: relative;
   grid-row: span 3;
   display: flex;
   align-items: center;
   justify-content: center;
   border-radius: 999em;
 }
+
+.dx-chat-message-avatar-image {
+  position: absolute;
```

Comment:
Why should we use absolute positioning for this element?

Reply:
[4da065f777adfedddf3bf741d68b4ba898273434]